UPGRADE.md

```diff
@@ -106,6 +106,10 @@ or any public API on instantiated objects.
 Please implement the `Doctrine\ORM\Repository\RepositoryFactory` interface instead of extending
 the `Doctrine\ORM\Repository\DefaultRepositoryFactory`.
 
+## BC BREAK: `Doctrine/\ORM\Persisters\BasicEntityPersister::getSelectConditionStatementColumnSQL` has been merked as `private`
+
+The `getSelectConditionStatementColumnSQL` has been removed (marked as `private`) from the API.
```

Comment:
We discussed [here](https://github.com/doctrine/doctrine2/pull/1113#discussion_r20154567) about this. If it is not required i can remove this.